tests/test_profile_management.py

```diff
@@ -0,0 +1,386 @@
+                readers=['openreview.net/Support', '~John_Last1'],
+                writers=['openreview.net/Support'],
+                signatures=['~John_Last1'],
+                content={
+                    'username': '~John_Last1',
+                    'comment': 'typo in my name',
+                    'status': 'Pending'
+                }
+            ))        
+
+
+        ## Add publications
+        john_client.post_note(openreview.Note(
+            invitation='openreview.net/Archive/-/Direct_Upload',
+            readers = ['everyone'],
+            signatures = ['~John_Alternate_Last1'],
+            writers = ['~John_Alternate_Last1'],
+            content = {
+                'title': 'Paper title 1',
+                'abstract': 'Paper abstract 1',
+                'authors': ['John Alternate Last', 'Test Client'],
+                'authorids': ['~John_Alternate_Last1', 'test@mail.com']
+            }
+        ))
+
+        john_client.post_note(openreview.Note(
+            invitation='openreview.net/Archive/-/Direct_Upload',
+            readers = ['everyone'],
+            signatures = ['~John_Alternate_Last1'],
+            writers = ['~John_Alternate_Last1'],
+            content = {
+                'title': 'Paper title 2',
+                'abstract': 'Paper abstract 2',
+                'authors': ['John Last', 'Test Client'],
+                'authorids': ['~John_Last1', 'test@mail.com']
+            }
+        ))
+
+        publications = client.get_notes(content={ 'authorids': '~John_Last1'})
+        assert len(publications) == 2
+
+        request_note = john_client.post_note(openreview.Note(
+            invitation='openreview.net/Support/-/Profile_Name_Removal',
+            readers=['openreview.net/Support', '~John_Last1'],
+            writers=['openreview.net/Support'],
+            signatures=['~John_Last1'],
+            content={
+                'username': '~John_Alternate_Last1',
+                'comment': 'typo in my name',
+                'status': 'Pending'
+            }
+
+        ))
+
+        helpers.await_queue()
+
+        messages = client.get_messages(to='john@profile.org', subject='Profile name removal request has been received')
+        assert len(messages) == 1
+        assert messages[0]['content']['text'] == '''Hi John Last,
+
+We have received your request to remove the name "~John_Alternate_Last1" from your profile: https://openreview.net/profile?id=~John_Last1.
+
+We will evaluate your request and you will receive another email with the request status.
+
+Thanks,
+
+The OpenReview Team.
+'''
+
+        ## Accept the request
+        decision_note = client.post_note(openreview.Note(
+            referent=request_note.id,
+            invitation='openreview.net/Support/-/Profile_Name_Removal_Decision',
+            readers=['openreview.net/Support'],
+            writers=['openreview.net/Support'],
+            signatures=['openreview.net/Support'],
```

Comment:
should the signature be tilde id of su?

Reply:
I think only super user is signatory of the support group so it is fine. I would like to stop using su for everything specially moderation that we would like to delegate the task to someone else.